src/main/resources/static/newDemo.js

```diff
@@ -1,72 +0,0 @@
-Vue.component('newDemo', {
-    name: "newDemo",
-    template: `
-        <v-form v-model="valid">
-            <v-container>
-              <v-layout>
-                <v-flex
-                  xs12
-                  md4
-                >
-                  <v-text-field
-                    v-model="firstname"
-                    :rules="nameRules"
-                    :counter="10"
-                    label="First name"
-                    required
-                  ></v-text-field>
-                </v-flex>
-        
-                <v-flex
-                  xs12
-                  md4
-                >
-                  <v-text-field
-                    v-model="lastname"
-                    :rules="nameRules"
-                    :counter="10"
```

Comment:
I don't really think that a counter should be used here. People have long names.
Same applies to first name